src/runtime/vm/profiler/vm.cc

```diff
@@ -45,7 +45,15 @@ PackedFunc VirtualMachineDebug::GetFunction(const std::string& name,
     return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
       ICHECK_EQ(args.size(), 1U);
       std::vector<std::pair<Index, double>> op_acc_time;
+      std::unordered_map<Index, std::vector<double>> op_durations;
       for (auto kv : op_durations_) {
+        std::vector<double> durations;
+        for (auto t : kv.second) {
+          durations.push_back(t.SyncAndGetTime() / 1e3);
```

Comment:
I think you want to cast to double before dividing

Reply:
`1e3` is double so this will always be a double.